core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -586,3 +593,23 @@ <h3 ng-if="action === 'publicize_exploration'">Boost exploration ranking (as mod
     <button class="btn btn-danger" ng-click="reallyDelete()">Delete Exploration</button>
   </div>
 </script>
+
+<script type="text/ng-template" id="modals/previewSummaryTile">
+  <div class="modal-header">
+    <h3>A preview for your summary tile.</h3>
```

Comment:
I asked around and my friends were split between summary panel and summary card. I've gone ahead and opted for summary card if that's alright.